packages/jest-emotion/test/printer.test.js

```diff
@@ -94,15 +100,23 @@ describe('jest-emotion with DOM elements disabled', () => {
 })
 
 test('does not replace class names that are not from emotion', () => {
-  const classes = cx(
-    'net-42',
-    'net',
-    css`
-      color: darkorchid;
-    `
-  )
-
-  let tree = renderer.create(<div className={classes} />).toJSON()
+  let tree = renderer
+    .create(
+      <ClassNames>
+        {({ css: classNamesCss, cx }) => (
+          <div
+            className={cx([
+              'net-42',
+              'net',
+              classNamesCss`
+                color: darkorchid;
```

Comment:
This doesn’t need to use ClassNames, you can use the css prop here and pass a class name as well and they’ll get concatenated.